src/auth/token-verifier-util.ts

```diff
@@ -0,0 +1,89 @@
+/*!
```

Comment:
These can go in the token-verifier module like they used to. No need to introduce a new module.

Reply:
Some of the components `ID_TOKEN_INFO` and `SESSION_COOKIE_INFO` are `auth` specific. Instead of moving them to the `auth` module, I introduced the token verifier util module for better separation. Would you rather have them all in the `auth` module or back in the new `token-verifier` util module?

Reply:
Renamed back to `auth/token-verifier.ts`. I agree that it makes more sense now. Thanks!